Tiltfile

```diff
@@ -468,17 +469,16 @@ def deploy_clusterclass(clusterclass_name, provider, filename):
         text = "Delete ClusterClass",
     )
 
-def deploy_cluster_template(template_name, provider, filename):
-    apply_cluster_template_cmd = "CLUSTER_NAME=" + template_name + "-$RANDOM; " + clusterctl_cmd + " generate cluster $CLUSTER_NAME --from " + filename + " | " + kubectl_cmd + " apply -f - && echo \"Cluster '$CLUSTER_NAME' created, don't forget to delete\n\""
-
+def deploy_cluster_template(template_name, label, filename, substitutions):
+    apply_cluster_template_cmd = clusterctl_cmd + " generate cluster $CLUSTER_NAME --from " + filename + " | " + kubectl_cmd + " apply -f - && echo \"Cluster '$CLUSTER_NAME' created, don't forget to delete\n\""
     delete_clusters_cmd = 'DELETED=$(echo "$(bash -c "' + kubectl_cmd + ' get clusters --no-headers -o custom-columns=":metadata.name"")" | grep -E "^' + template_name + '-[[:digit:]]{1,5}$"); if [ -z "$DELETED" ]; then echo "Nothing to delete for cluster template ' + template_name + '"; else echo "Deleting clusters:\n$DELETED\n"; echo $DELETED | xargs -L1 ' + kubectl_cmd + ' delete cluster; fi; echo "\n"'
```

Comment:
Nit: we could probably change this regex `grep -E "^' + template_name + '-[[:digit:]]{1,5}$")`. Before, `$RANDOM` would give us a number between 0 and 32767 so the regex selects any number with 0-5 digits. But now maybe we can just select any number after the template name.